tox.ini

```diff
@@ -1,22 +1,22 @@
 [tox]
 envlist =
     check
-    py37,
     py38,
+    py39,
     coverage
     docs
 
 [testenv]
 usedevelop = True
 extras = test
+deps = Cython
```

Comment:
Why do we need `Cython` here? (Oh maybe this was from the example I showed you? In that case it was necessary for that specific thing)

Reply:
Should I add back the `basepython = python3` statements I removed?

Reply:
hmm possibly, or else when running on a local machine it would run `python` by default, which might still point to python 2